pkg/sensors/program/loader.go

```diff
@@ -423,6 +423,9 @@ func TracingAttach() AttachFunc {
 
 func LSMOpen(load *Program) OpenFunc {
 	return func(coll *ebpf.CollectionSpec) error {
+		delete(coll.Programs, "ima_file_open")
+		delete(coll.Programs, "ima_mmap_file")
+		delete(coll.Programs, "ima_bprm_check_security")
```

Comment:
Why do we remove the programs here?

Reply:
Because, we can't load them all (they are not generic). We need to load one of them only if `imaHash: true` in the policy. I load ima_* programs on demand, after all lsm generic programs are loaded.